packages/app/src/debug/DebugPendingRunCounts.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <span
+    data-cy="debug-pending-counts"
+    class="text-gray-600"
+  >
+    {{ t('debugPage.pending.pendingCount', { n: totalTests, completed: totalCompleted, total: totalTests }) }}
+  </span>
+</template>
+
+<script setup lang="ts">
+import { gql } from '@urql/core'
+import { computed } from '@vue/reactivity'
+import { useI18n } from 'vue-i18n'
+
+gql`
+fragment DebugPendingRunCounts on CloudRun {
+  id
+  totalSkipped
+  totalPassed
+  totalFailed
+  totalTests
```

Comment:
Same comment from the `DebugNewRelevantRunBar`:
These fields are counts of tests and this component should be reflecting counts of specs.  Right now the Cloud is not providing this data summarized so you would have to get the `specs` field from the `CloudRun` and group by the `status` field.

Reply:
Addressed in [`822015a`](https://github.com/cypress-io/cypress/pull/25282/commits/822015a319f0403c2c1ee5303226c9764227ad83)